engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropReflectionNode.java

```diff
@@ -50,7 +50,6 @@
  *
  */
 public class BackdropReflectionNode extends AbstractNode {
-
```

Comment:
I usually do it manually.. Should I be avoiding that? I do it mostly to maintain consistency between files.